packages/windows-launcher/default.nix

```diff
@@ -143,14 +143,14 @@
         fi
 
         if ! ${windowsLauncher} $FILE; then
-          ${gnome.zenity}/bin/zenity --error --text="Failed to run Windows VM: $?"
+          ${yad}/bin/yad --error --text="Failed to run Windows VM: $?"
```

Comment:
Thanks, that is a nice tool. I didn't know about it. I can't test it at the moment but it looks like there is no --error parameter. I guess it should be replaced with --image=gtk-dialog-error.